HypeBears.sol

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.4;
+
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/IERC721Metadata.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/utils/Context.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/IERC721Enumerable.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+
+
+contract HypeBears is ERC721("HypeBears", "HB"), ERC721Enumerable, Ownable {
+    using ECDSA for bytes32;
+    using SafeMath for uint256;
+    using Strings for uint256;
+
+    address public proxyRegistryAddress = 0xa5409ec958C83C3f309868babACA7c86DCB077c1;
+
+    string private baseURI;
+    string private blindURI;
+    uint256 public mintLimit = 1;
+    uint256 private constant TOTAL_NFT = 10000;
+    uint256 public mintPrice = 0.4 ether;
```

Comment:
These storage vars not being constant allows owners flexibility to modify the values later on. If there's no intent to do so, which 99% of the time there isn't, not having them as const means SLOAD operations are required, costing ~5k gas